integration/src/test/java/org/mule/test/integration/exceptions/ErrorHandlerLifecycleTestCase.java

```diff
@@ -49,6 +51,14 @@ protected String getConfigFile() {
   @Named("flowD")
   private FlowConstruct flowD;
 
+  @Inject
+  @Named("flowE")
+  private FlowConstruct flowE;
+
+  @Inject
+  @Named("flowF")
+  private FlowConstruct flowF;
+
   @Test
   public void testLifecycleErrorHandlerInFlow() throws Exception {
```

Comment:
verify in this tests also that there are no double application of phases. I think you can set a default property of the error tracker for this.